seed/static/seed/js/controllers/inventory_detail_sensors_controller.js

```diff
@@ -175,6 +175,41 @@ angular.module('BE.seed.controller.inventory_detail_sensors', [])
         fastWatch: true
       };
 
+      $scope.exportSensors = function (grid_data, data_type) {
+        let title = `${data_type}_for_${$scope.inventory_type == 'properties' ? 'Property' : 'Taxlot'}_${$scope.item_state.pm_property_id}_${moment().format('YYYY_MM_DD')}`;
+        let keys = grid_data.columnDefs.map(c => c.field).filter(key => key != 'data_logger');
+        let data = [keys.join(',')];
+
+        grid_data.data.forEach(d => {
+          let row = [];
+          keys.forEach(k => row.push(d[k]));
+          data.push(row.join(','));
+        });
+
+        saveAs(new Blob([data.join('\n')], {type: 'text/csv'}), title);
+      }
+      $scope.exportSensorUsages = function (grid_data, sensor_data, data_type) {
+        let title = `${data_type}_for_${$scope.inventory_type == 'properties' ? 'Property' : 'Taxlot'}_${$scope.item_state.pm_property_id}_${moment().format('YYYY_MM_DD')}`;
+        let keys = grid_data.columnDefs.map(c => c.field)
+
+        // grid data headers are a combination of display name and data logger. Use sensor_data to make a conversion object
+        let display_to_column = {'timestamp': 'timestamp'}
+        sensor_data.data.forEach(sensor => {
+          display_to_column[`${sensor.display_name} (${sensor.data_logger})`] = sensor.column_name
+        })
```

Comment:
To allow backward compatibility, the headers need to match `sensor.column_name`.